guides/advanced_guides/faceted_search.md

```diff
@@ -0,0 +1,49 @@
+# Faceted Search
+
+Faceted search is a feature provided out-of-the-box by MeiliSearch. Faceting allows to classify search results into categories that are called **facets**.
+
+> For a movie, its title, director, and its release date can be used as facets.
+
+The user can then narrow down search results by selecting facets, which is similar to applying filters to refine the results.
+
+Both faceting and filtering help narrow down the search results. However, **faceting differs from [filtering](/guides/advanced_guides/filtering.md)**.
+
+Faceted attributes **must be declared at indexing time**.
+
+## Using Facets
```

Comment:
```suggestion
## Setting Up Facets
```

Reply:
Cannot commit this suggestion so I add it locally.